### PR TITLE
Azaya's beautiful zarr contribution.

### PR DESCRIPTION
**Pull request by Azaya89**
This is my contribution to the Beautiful Zarr project containing Zarr use cases, code snippets and visualization, and Twitter mentions of Zarr.

### CODE REVIEW

File: _data/azaya89/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- https://www.youtube.com/watch?v=oWltJsPlIaQ&t=173s
+
+----
+
+
+## Code Snippets and visualisation:
+
+
+- [Interactive design of GPU-accelerated Image Data Flow Graphs and cross-platform deployment using multi-lingual code generation](https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/32_tiled_image_processing/tiled_image_file_formats_zarr.html)
```
Comment (sanketverma1704) [inline _data/azaya89/README.md]: The title and the link to the code don't match. Can you please correct the title?

**Comment by sanketverma1704** [comment]
Hi @Azaya89. Thanks for sending the PR.
I've left a comment, please have a look at it.

File: _data/azaya89/README.md (hunk 1 above)
Comment (Azaya89) [reply _data/azaya89/README.md]: I have resolved it now. Thank you for pointing it out.

**Comment by Azaya89** [comment]
> Hi @Azaya89. Thanks for sending the PR. I've left a comment, please have a look at it.

Done. Pls review, thank you!

**Comment by sanketverma1704** [comment]
Thanks for updating the PR @Azaya89. It looks good.
Merging this now. Feel free to add more stuff as you go.

**Pull request merged**